schematron/v2.2.0/exemplary_files/L200_Audit.xml

```diff
@@ -744,10 +751,21 @@
           <auc:CustomMeasureName>Low Cost Measure Name</auc:CustomMeasureName>
           <!-- 6.1.5.a -->
           <auc:LongDescription>This measure does something cheap</auc:LongDescription>
+          <auc:Recommended>true</auc:Recommended>
+          <auc:StartDate>2020-01-01</auc:StartDate>
+          <auc:EndDate>2020-12-01</auc:EndDate>
         </auc:Measure>
 
         <!-- Capital measure -->
         <auc:Measure ID="Measure-Capital-A">
+          <auc:TypeOfMeasure>
+            <auc:Additions>
+              <auc:Addition>
+                <auc:ExistingScheduleAffected IDref="PlugLoadSchedule" />
```

Comment:
We should create some measures that would be real 'capital' measures.  We can add better measures later - maybe mark this as an issue / something to track making this better?

Reply:
Yes I'll add that, I haven't made it that far yet, hope to get that in before this PR is merged

Reply:
Created this issue here for improving exemplary files: #45